cpp/ps0007/p0007.soln.cpp

```diff
@@ -53,9 +56,13 @@ int main() {
             cout << "Invalid shape type." << endl;
             return -1;
     }
+
+    // check the validaty of the choice for the shape 1/1
+    // printing out the area of the shape 1/1
     double result = area(shape, dimensions);
     if (result != -1) {
         cout << "The area is: " << result << endl;
     }
     return 0;
 }
+// great work 😎 9.5/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors while compilation & no bugs or warnings while execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code
- [x] used array & function concepts for the program
- [ ] good design for function using DRY concept
- [x] main returns non-zero/zero number according to the case
+0.5 bonus mark for your effort ⭐

you can check the [problem solution](https://onlinegdb.com/wybZ67yJX) for more